meerk40t/gui/navigationpanels.py

```diff
@@ -487,6 +527,59 @@ def on_button_align_trace_quick(
         self.context("trace_quick\n")
         self.drag_ready(True)
 
+    def pane_show(self, *args):
+        self.context.listen("driver;position", self.on_update)
+        self.context.listen("emulator;position", self.on_update)
+
+    # Not sure whether this is the right thing to do, if it's still locked and then
+    # the pane gets hidden?! Let's call it a feature for now...
+    def pane_hide(self, *args):
+        self.context.unlisten("driver;position", self.on_update)
+        self.context.unlisten("emulator;position", self.on_update)
```

Comment:
Emulator position refers to the `lhyemulator` and I gotta imagine if you don't turn this off before starting a job it will go pretty crazy if the driver position is still locked. The job will be set because it'll already be through the planner but that would look pretty interesting to have it updating a lot based on the laser head position.